### PR TITLE
registering demoparticipant

### PR DESCRIPTION
**Pull request by gauravmishra-17**
Hi maintainer/s!!
You are/ all are doing great job!
I hope I am not late and my PR gets accepted. Looking forward for the journey.

### CODE REVIEW

File: content/participants/ttjc@gauravmishra-17/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+I can be too much practical person, where everyone things I am rude.
+
+### Help I need?
+
+I just want someone to help me understand the flowchart of building a project. I get lost to easily.
+
+### How to contact me?
+
+- Twitter
```
Comment (khusharth) [inline content/participants/ttjc@gauravmishra-17/index.md:32]: Hey! @gauravmishra-17 please add your twitter link here and if you don't have one then remove it.

Comment (gauravmishra-17) [reply content/participants/ttjc@gauravmishra-17/index.md]: @khusharth I have added the twitter link

Comment (gauravmishra-17) [reply content/participants/ttjc@gauravmishra-17/index.md]: Hello maintainers!? will my pr get merged? Please accept my pr. I edited my line as it was suggested to be done before the deadline.

Comment (gauravmishra-17) [reply content/participants/ttjc@gauravmishra-17/index.md]: @khusharth  @gkanishk

Comment (gkanishk) [reply content/participants/ttjc@gauravmishra-17/index.md]: Chill man!!
Reviewing PR takes time, should have patience.

Comment (gauravmishra-17) [reply content/participants/ttjc@gauravmishra-17/index.md]: man I was freaked about the deadline.

**Pull request merged**